addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3171,26 +3229,35 @@ class SnippetsMenu extends Component {
         };
 
         this.snippets.clear();
+        let index = 0;
         for (const snippetsEl of html.querySelectorAll("snippets")) {
             const category = {
                 id: snippetsEl.id,
                 text: snippetsEl.getAttribute("string"),
                 classes: [...snippetsEl.classList]
             };
             for (const snippetEl of snippetsEl.children) {
+                const isCustom = !!snippetEl.closest('#snippet_custom');
                 const snippet = {
                     id: parseInt(snippetEl.dataset.oeSnippetId) || uniqueId(snippetEl.dataset.moduleId),
                     name: snippetEl.children[0].dataset.snippet,
-                    displayName: escape(snippetEl.getAttribute("name")),
+                    displayName: markup(escape(snippetEl.getAttribute("name"))),
```

Comment:
check with security maybe ? without that the "Contact & Froms" group is displayed with `&amp;` instead of `&`